components/ResultsTable.go

```diff
@@ -1224,3 +1265,52 @@ func (table *ResultsTable) search() {
 
 	table.SetInputCapture(nil)
 }
+
+func (table *ResultsTable) ShowSidebar(show bool) {
+	table.state.showSidebar = show
+
+	if show {
+		table.UpdateSidebar()
+		table.Page.SendToFront(SidebarPageName)
+		table.Page.ShowPage(SidebarPageName)
+	} else {
+		table.Page.HidePage(SidebarPageName)
+		App.SetFocus(table)
+	}
+}
+
+func (table *ResultsTable) UpdateSidebar() {
+	columns := table.GetColumns()
+	selectedRow, _ := table.GetSelection()
+
+	if selectedRow > 0 {
+		tableX, _, _, tableHeight := table.GetRect()
+		_, _, tableInnerWidth, _ := table.GetInnerRect()
+		_, tableMenuY, _, tableMenuHeight := table.Menu.GetRect()
+		_, _, _, tableFilterHeight := table.Filter.GetRect()
+
+		sidebarWidth := (tableInnerWidth / 4)
+
+		table.Sidebar.SetRect(tableX+tableInnerWidth-sidebarWidth, tableMenuY, sidebarWidth, tableHeight+tableMenuHeight+tableFilterHeight)
+		table.Sidebar.Clear()
+
+		for i := 1; i < len(columns); i++ {
+			name := columns[i][0]
+			colType := columns[i][1]
+
+			text := table.GetCell(selectedRow, i-1).Text
+			title := name
+
+			logger.Info("string repeat", map[string]any{"sidebarWidth": sidebarWidth, "name": len(name), "colType": len(colType)})
+			repeatCount := sidebarWidth - len(name) - len(colType) - 6 // 2 for spaces added below and idk why 6 is needed, but it works.
+
+			if repeatCount <= 0 {
+				repeatCount = 1
+			}
+			title += "[gray]" + strings.Repeat("-", repeatCount)
```

Comment:
Shouldn't gray be a constant you define somewhere to provide a consistent theming